awesomemotive/affiliatewp-affiliate-area-shortcodes
# issue/41 - Bump WordPress minimum requirements

Issue #41 


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate Affiliate Area Shortcodes add-on
2. Confirm the plugin works as expected without any PHP errors.
3. Create a new page with the shortcode: `[affiliate_area]` and confirm the page shows the affiliate area
